src/Benchmarks/Middleware/CopyToAsyncMiddleware.cs

```diff
@@ -20,17 +21,25 @@ public CopyToAsyncMiddleware(RequestDelegate next)
             _next = next;
         }
 
-        public Task Invoke(HttpContext httpContext)
+        public async Task Invoke(HttpContext httpContext)
         {
             if (httpContext.Request.Path.StartsWithSegments(_path, StringComparison.Ordinal))
             {
                 httpContext.Response.StatusCode = 200;
                 httpContext.Response.ContentType = "text/plain";
 
-                return httpContext.Request.Body.CopyToAsync(httpContext.Response.Body);
+                using (var ms = new MemoryStream())
+                {
+                    // Don't copy to response stream, so this measures just
+                    // the time to copy the body and not copying plus writing
+                    // the response.
+                    await httpContext.Request.Body.CopyToAsync(ms);
+                }
+
+                await httpContext.Response.WriteAsync("Hello World!");
             }
 
-            return _next(httpContext);
+            await _next(httpContext);
```

Comment:
Very weird to call `_next` after writing to the `Response`. This doesn't match what the test did before.